inline/tests/spec/snapshots/parser/bold/escaped-bold.snap

```diff
@@ -1,9 +1,10 @@
 ---
 source: inline/tests/parser/mod.rs
-info: "Test 'escaped-bold' from: 'bold.yml'"
+info: "Test 'escaped-bold' from 'bold.yml'"
 ---
 Plain @ (1:1)->(1:17) (
-    **not bold**
+    ␢␢**not bold**␢␢
```

Comment:
```suggestion
    ␢*␢*not bold*␢*␢
```

Would it be difficult to render it like this?
Just to show where a backslash was actually set.
It would also be consistent with the lexer snapshot.

Reply:
Unfortunately yes. Depending on the test case maybe even completely impossible. 
This is because at this point we only have a single `String` of content, and no information where the `Span` in input and output length mismatch happened.

Reply:
no problem :)
Lets leave it as is then